soc/xtensa/nxp_adsp/imx8m/linker.ld

```diff
@@ -169,6 +169,16 @@ PROVIDE(_memmap_cacheattr_reset = _memmap_cacheattr_imx8_wt_allvalid);
 _EXT_MAN_ALIGN_ = 16;
 EXTERN(ext_man_fw_ver)
 
+SECTIONS
+{
+#ifdef CONFIG_OPENAMP_RSC_TABLE
+    SECTION_PROLOGUE(.resource_table,, SUBALIGN(4))
+    {
+      KEEP(*(.resource_table*))
+    } GROUP_LINK_IN(ROMABLE_REGION)
+#endif
+ }
```

Comment:
Why adding a new `SECTIONS` instead of using the one at line 182?

Reply:
Done